pages/index.tsx

```diff
@@ -1,18 +1,18 @@
+import React from "react";
```

Comment:
Again, is it neccesary?

Reply:
Eslint throws an error for react import, why I'm importing it

Reply:
Okay cool.